src/groupeddatatable/grouping.jl

```diff
@@ -355,16 +355,23 @@ dt |> groupby(:a) |> [sum, x->mean(dropnull(x))]   # equivalent
 """
 aggregate(d::AbstractDataTable, fs::Function; sort::Bool=false) = aggregate(d, [fs], sort=sort)
 function aggregate{T<:Function}(d::AbstractDataTable, fs::Vector{T}; sort::Bool=false)
```

Comment:
> When a user specifies sort = true, groupby will sort on the grouped columns and then aggregate only bothers to sort the remaining columns that weren't sorted during the groupby.

I must be missing something. If `aggregate` does not sort again on the grouped columns (which is a no-op when they are already sorted), the result won't be sorted on these columns.

Anyway if that works it's fine to keep the existing strategy (though if it can be improved in another PR, why not).